src/QsCompiler/Tests.Compiler/TestCases/PopulateCallGraph.qs

```diff
@@ -307,3 +388,75 @@ namespace Microsoft.Quantum.Testing.PopulateCallGraph {
         Bar(Foo, x);
     }
 }
+
+// =================================
+
+// Concrete Graph Multiple Entry Points
+namespace Microsoft.Quantum.Testing.PopulateCallGraph {
+
+    @EntryPoint()
+    operation Main1() : Unit {
+        Foo<Double>();
+    }
+
+    @EntryPoint()
+    operation Main2() : Unit {
+        Foo<String>();
+    }
+
+    operation Foo<'A>() : Unit { }
+}
+
+// =================================
+
+// Concrete Graph Unit Tests
+namespace Microsoft.Quantum.Testing.PopulateCallGraph {
+
+    open Microsoft.Quantum.Diagnostics;
+
+    @Test("QuantumSimulator")
+    operation Test1() : Unit {
+        Foo<Double>();
+    }
+
+    @Test("QuantumSimulator")
+    operation Test2() : Unit {
+        Foo<String>();
+    }
+
+    operation Foo<'A>() : Unit { }
+}
+
+// =================================
+
+// Concrete Graph Entry Points and Unit Tests
+namespace Microsoft.Quantum.Testing.PopulateCallGraph {
+
+    open Microsoft.Quantum.Diagnostics;
+
+    @EntryPoint()
+    operation Main1() : Unit {
+        Foo<String>();
+    }
```

Comment:
(I'm not confident in .qs, my note may be mistaken)

(At least for C/C++) It is suspicious to see the same name (`Main1`) for different function definitions in the same namespace. One definition is
```qsharp
    @EntryPoint()
    operation Main1() : Unit {
        Foo<Double>();
    }
```
another one is
```qsharp
    @EntryPoint()
    operation Main1() : Unit {
        Foo<String>();
    }
```
If some fragment calls `Main1()` then which definition will be called?

Reply:
This file with Q# test cases is separated into chunks by the lines with `// =================================`. This is done by the unit test driving logic. Each chunk is compiled totally independent of the other chunks. That is to say that each test case is isolated from the other test cases. In the test case that you linked to above, there is an operation named `Main1` for the first entry point, and an operation for `Main2` for the second entry point. If they are named the same, then that is an accident and is incorrect. Q# does not allow multiple callables with the same name in the same namespace when it compiles the code.

Reply:
Based on your other feedback, this is not applicable any more. Resolving.